jujugui/static/gui/src/app/store/env/go.js

```diff
@@ -476,25 +476,32 @@ YUI.add('juju-env-go', function(Y) {
     handleEnvironmentInfo: function(data) {
       if (data.Error) {
         console.warn('Error retrieving environment information.');
+        return;
+      }
+      var self = this;
```

Comment:
Why are you creating self? I don't see a place where the context for `this` changes.